core/tee/tee_fs_rpc.c

```diff
@@ -23,6 +24,27 @@ struct tee_fs_dir {
 	struct tee_fs_dirent d;
 };
 
+/* "/dirf.db" or "/<file number>" */
+static TEE_Result
+tee_svc_storage_create_filename_dfh(void *buf, size_t blen,
```

Comment:
Would it make sense to rename this function to `create_filename()` also?

Reply:
Absolutely.